app/src/main/java/org/p2p/wallet/auth/widget/SearchView.kt

```diff
@@ -0,0 +1,101 @@
+package org.p2p.wallet.auth.widget
+
+import android.animation.Animator
+import android.content.Context
+import android.widget.RelativeLayout
+import android.view.ViewAnimationUtils
+import android.text.TextWatcher
+import android.util.AttributeSet
+import androidx.core.view.isVisible
+import org.p2p.uikit.utils.focusAndShowKeyboard
+import org.p2p.uikit.utils.hideKeyboard
+import org.p2p.wallet.databinding.WidgetSearchViewBinding
+import org.p2p.wallet.utils.viewbinding.inflateViewBinding
+
+class SearchView @JvmOverloads constructor(
```

Comment:
please find a different name for this widget, it's not okay to conflict with Android names